### PR TITLE
drones controller added + index/new/show views

### PR DESCRIPTION
**Pull request by SabrinaNayeli**
generated the drones controllers, made the index new and show views. I also added the validates for the drone model

### CODE REVIEW

File: app/views/drones/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>Drones</h1>
+
+<% @drones.each do |drone| %>
```
Comment (Tom-Firth) [inline app/views/drones/index.html.erb:3]: Je sais pas si tu as fini cette view ou pas, dans le doute, il te manques un <% end %>

Comment (SabrinaNayeli) [reply app/views/drones/index.html.erb:3]: corrected

File: app/views/drones/index.html.erb [hunk 2]
```diff
@@ -4,3 +4,4 @@
 <div>
   <p><%= drone.name %></p>
```
Comment (AndyLukac) [inline app/views/drones/index.html.erb]: Just an idea.
On our cards (home index), we should display more info than just the drone.name.

For example we can add stars for reviews or the name of the owner.

More ideas are welcome of course!

Comment (SabrinaNayeli) [reply app/views/drones/index.html.erb]: I totally agry. Will add this

**Pull request merged**